### DISCUSSION TITLE
Remove the blue stripe from the bottom

### DISCUSSION THREAD

**Discussion by C1eriC** [valinet/ExplorerPatcher#211] [Q&A]
Hello! Thanks a lot for big work,
How can I remove the [blue stripe from the bottom](https://habrastorage.org/webt/yc/pq/_y/ycpq_ydlmji6u0obvnpk2tuwf0a.png)?

**Comment by valinet** [comment]
Hi

I don’t know exactly. Maybe a custom theme?

Thanks

**Comment by rxboy1983** [comment]
**C1eriC**,
use app AccentPalette Tool for change color.
Download here: https://github.com/Raymai97/AccentPaletteTool/releases/tag/v1.
![Zrzut ekranu 2021-10-26 180835](https://user-images.githubusercontent.com/93114065/138919096-ce26eebb-f9d3-401a-8b40-466c67b46992.png)
1

**Comment by Gaurav-Original-ClassicShellTester** [comment]
In case you are ok with an affordable commercial app, StartIsBack/StartAllBack lets you use visual styles/msstyles for the taskbar that are independent of Windows msstyles (thus surviving major Windows build upgrades). It already has Windows 7 and 8.1 styles built-in without the underlines, plus custom ones you can make for yourself using Windows Style Builder, and custom fan-made ones like these: https://www.deviantart.com/sagorpirbd/art/XP-Styles-Pack-for-Win10-587254466

![image](https://user-images.githubusercontent.com/46422834/139025934-004c76af-2178-4046-b8f7-6cb54e562645.png)
